components/menu.module.css

```diff
@@ -1 +1,167 @@
-/* the style for the menu goes here */
+.hamburger {
+  position: fixed;
+  z-index: 1000;
+  top: 3.5rem;
+  left: 2.5rem;
+	width: 3rem;
+  height: 2.2rem;
+  cursor: pointer;
+}
+
+.hamburger_line {
+  position: absolute;
+  top: 0;
+  left: 0; 
+  width: 3rem;
+  height: 0.2rem;
+  background-color: var(--accent-color);
+}
+
+.hamburger_line::before {
+  content: '';
+  position: absolute;
+  top: 0.8rem;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: var(--accent-color);
+}
+
+.hamburger_line::after {
+  content: '';
+  position: absolute;
+  top: 1.6rem;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: var(--accent-color);
+}
+
+.cross_line {
+  position: absolute;
+  top: 0.5rem;
+  left: 0;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: black;
+  transform: rotate(45deg);
+}
+
+.cross_line::after {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 3rem;
+  height: 0.2rem;
+  background-color: black;
+  transform: rotate(90deg);
+}
+
+.menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 70%;
+  height: 100vh;
+  background-color: var(--accent-color);
+  transition: all 0.5s ease-in-out;
+  z-index: 100;
+  padding: 8rem 5rem;
+}
+
+.menu_hidden {
+  position: fixed;
+  top: 0;
+  left: -100vw;
+  width: 70%;
+  height: 100vh;
+  background-color: var(--accent-color);
+  z-index: 100;
+  padding: 8rem 5rem;
+  transition: all 0.5s ease-in-out;
+}
+
+.menu ul, .menu_hidden ul {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+}
+
+.menu li, .menu_hidden li {
+  font-family: var(--title-font);
+  font-size: 2rem;
+  text-decoration: none;
+  border: none;
+  list-style: none;
+}
```

Comment:
```suggestion
  list-style: none;
  transition: color 0.1s ease;
  display: inline-block;
}
```
I realized that the color snaps and changes its colour, so I suggest putting `transition` like above.
Plus, I don't want each item to have a redundant/longer underline so I put `display: inline-block`.

It works like this:
![menu-hover-actions](https://user-images.githubusercontent.com/51708229/145870947-02790ca1-9e70-4f78-aa9c-bfcd4ce03bef.gif)

Reply:
Agreed!